crates/matrix-sdk/src/event_handler.rs

```diff
@@ -269,17 +291,25 @@ impl Client {
             unsigned: Option<UnsignedDetails>,
         }
 
+        // FIXME: Iterate the events only once
```

Comment:
Same: you can leave a comment or a note, but a FIXME should go into the issue tracker.

Reply:
Okay, let's leave this thread open then and I'll use it as a reminder to create that issue after the PR is merged.